validator/client/validator.go

```diff
@@ -133,7 +133,7 @@ func (v *validator) WaitForChainStart(ctx context.Context) error {
 	// First, check if the beacon chain has started.
 	stream, err := v.validatorClient.WaitForChainStart(ctx, &ptypes.Empty{})
 	if err != nil {
-		return errors.Wrap(err, "could not setup beacon chain ChainStart streaming client")
+		return errors.Wrap(err, "could not setup beacon chain ChainStart streaming client"+connectionIssueMsg)
```

Comment:
maybe there could be a cleaner way to wrap the connection error here. You could have a small function which would wrap the underlying error with the connection error for all the cases.

Reply:
fixed please take another look